conf/messages.en

```diff
@@ -367,6 +367,10 @@ membersSummary.event2.table.header2 = National Insurance number
 membersSummary.event2.table.header3 = Amount paid
 membersSummary.event2.table.totalTitle = Total not required
 
+membersSummary.search.title = Search for a member
+membersSummary.search.hint = Search by name or National Insurance number
+membersSummary.search = Search
+
```

Comment:
Do we need to add in a `.hidden` key for accessibility purposes and then add to corresponding view?

Reply:
On discussion this is not present for other similar inputs so closing thread.